src/prefect/cli/_prompts.py

```diff
@@ -162,18 +162,34 @@ async def find_flow_functions_in_file(path: anyio.Path) -> list[dict[str, str]]:
     return decorated_functions
 
 
-async def search_for_flow_functions(directory: str = ".") -> list[dict[str, str]]:
+async def search_for_flow_functions(
+    directory: str = ".", exclude_patterns: list[str] | None = None
+) -> list[dict[str, str]]:
     """
     Search for flow functions in the provided directory. If no directory is provided,
     the current working directory is used.
 
+    Args:
+        directory: The directory to search in
+        exclude_patterns: List of patterns to exclude from the search.
```

Comment:
Since the default logic is within this function I think it makes sense to keep the default in the docstring - sorry if my message was confusing earlier!